packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.html

```diff
@@ -1,12 +1,29 @@
 <template>
-    <li class="slds-m-horizontal--small slds-grid">
+    <li class="slds-var-m-horizontal_medium slds-grid">  
         <div class="slds-col_bump-right" onclick={onSelect}>
-            <button class="slds-m-horizontal_medium nowrap slds-button slds-text-color_default">
-                {item.value}
-                <span class="slds-m-horizontal_small">
-                    ({item.numberOfResults})
-                </span>
-            </button>
+            <template if:false={withsearch}>
+                <button class="slds-var-m-horizontal_medium nowrap slds-button slds-text-color_default facet__value">
+                    {item.value}
+                    <span class="facet__number-of-results">
+                        ({item.numberOfResults})
+                    </span>
+                </button>
+            </template>
+             <template if:true={withsearch}>
+                 <button class="slds-button slds-text-color_default">
+                    <div class="slds-grid slds-grid_vertical slds-gutters">
+                        <div class="slds-col facet__search-results">
+                            <lightning-formatted-rich-text class="slds-truncate slds-float_left" value={item.highlightedResult}></lightning-formatted-rich-text>
```

Comment:
I don't get it, shouldn't there be a click event somewhere here? If you search and display results, there should be a click somewhere to select the result no?

Reply:
I didn't get your comment